src/main/java/uk/gov/hmcts/et/common/model/ccd/CaseData.java

```diff
@@ -867,12 +868,15 @@ public class CaseData extends Et1CaseData {
     private String et3EmailLinkToAcas;
 
     //Referral
+    @JsonProperty("referralCollection")
+    private List<ReferralTypeItem> referralCollection;
     @JsonProperty("referralHearingDetails")
     private String referralHearingDetails;
+    //Referral Type
     @JsonProperty("referCaseTo")
     private String referCaseTo;
-    @JsonProperty("referentEmail")
-    private String referentEmail;
+    @JsonProperty("referrerEmail")
```

Comment:
more of a heads up than an issue. just be careful if you are changing the JSON property. If the property is live and in production it should never be changed as it can cause issues between the code and CCD (No field found error message)